content/swiftui/concepts/viewmodifier/terms/padding/padding.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: '.padding()'
+Description: 'Applies padding to the view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.padding()`** modifier method applies padding to the [view](https://www.codecademy.com/resources/docs/swiftui/views).
+
+## Syntax
+
+```pseudo
+Text("This text has padding around it.")
+    .padding(edges, amount)
+```
+
+The `.padding()` modifier method uses the following optional parameters:
+
+- The `edges` is a [set](https://www.codecademy.com/resources/docs/swift/sets) that specifies which side(s) the padding should be applied to. The following options are possible:
```

Comment:
The first parameter is not a set in the data structure sense, but a set as in option set. Let's remove the term "set" to avoid confusion and say:

```suggestion
- `edge` specifies which side(s) the padding should be applied to. The following options are possible:
```

Reply:
Thank you for the explanation!